torch/_export/serde/serialize.py

```diff
@@ -1305,12 +1305,13 @@ def deserialize_meta_func(serialized_target: str):
                 nn_module_stack[key] = (kv[key_idx + 2:comma_idx], module)
             ret["nn_module_stack"] = nn_module_stack
 
-        if source_fn_str := metadata.get("source_fn"):
+        if source_fn_st_str := metadata.get("source_fn_stack"):
             # Originally serializes to "fx_node_name,op_str"
-            source_fn = source_fn_str.split(",")
-            op = deserialize_meta_func(source_fn[1])
-            ret["source_fn"] = (source_fn[0], op)
-
+            source_fn_st = []
+            for source_fn_str in source_fn_st_str.split(";"):
+                name, target_str = source_fn_str.split(",")
+                source_fn_st.append((name, deserialize_meta_func(target_str)))
+            ret["source_fn_stack"] = source_fn_st
```

Comment:
I realized, this will break bc because existing serialized models will expect a singular source fn. So can you add some casing thats like "if previously it was a single source fn do xx else do this"

Reply:
yeah, that's a good point and many things will be broken e.g. the meta data name, etc. Maybe we should just fix these internal code and ignore the rest..  Any comments? @SherlockNoMad

Reply:
@angelayi could you point to the test? looks like it's not catched in OSS CI or sandcastle?